From: Y Giridhar Appaji Nag <giridhar@appaji.net>
Date: Sun, 27 Jan 2008 01:51:00 +0530
Subject: Change the bug reporting ID to submit@bugs.debian.org

Description: Change the bug reporting ID to submit@bugs.debian.org
Author: Y Giridhar Appaji Nag <giridhar@appaji.net>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
---
 configure.ac          | 2 +-
 src/Headers/version.h | 5 ++++-
 src/help.c            | 2 +-
 3 files changed, 6 insertions(+), 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index 515e60c..c11448f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,7 +1,7 @@
 dnl Process with autoconf to create a configure script  -*- Autoconf -*-
 AC_PREREQ(2.50)
 
-AC_INIT([Splint], [3.1.2], [splint-bug@splint.org], [splint])
+AC_INIT([Splint], [3.1.2], [submit@bugs.debian.org (via reportbug)], [splint])
 
 dnl This MUST precede any other macro
 AC_CONFIG_AUX_DIR([config])
diff --git a/src/Headers/version.h b/src/Headers/version.h
index c7c6cd7..7c9cabf 100644
--- a/src/Headers/version.h
+++ b/src/Headers/version.h
@@ -17,7 +17,10 @@
 # define SPLINT_LIBVERSION 3.018
 
 /*@constant observer char *SPLINT_MAINTAINER; @*/
-# define SPLINT_MAINTAINER "splint-bug@splint.org"
+# define SPLINT_MAINTAINER "submit@bugs.debian.org (via reportbug)"
+
+/*@constant observer char *SPLINT_DEBIAN_MAINTAINER; @*/
+# define SPLINT_DEBIAN_MAINTAINER "splint@packages.debian.org"
 
 # else
 # error "Multiple include"
diff --git a/src/help.c b/src/help.c
index d27602b..0487b62 100644
--- a/src/help.c
+++ b/src/help.c
@@ -274,7 +274,7 @@ printFlags (void)
 static void
 printMaintainer (void)
 {
-  llmsg (message ("Maintainer: %s", cstring_makeLiteralTemp (SPLINT_MAINTAINER)));
+  llmsg (message ("Maintainer: %s", cstring_makeLiteralTemp (SPLINT_DEBIAN_MAINTAINER)));
   llmsglit (LCL_COMPILE);
 }
 
